.eslintrc.json

```diff
@@ -12,6 +13,7 @@
   },
   "plugins": ["@typescript-eslint"],
   "rules": {
-    "no-console": "off"
+    "no-console": "off",
+    "@typescript-eslint/consistent-type-imports": "error"
```

Comment:
I'll drop a link to the docs on this rule as I looked it up: https://typescript-eslint.io/rules/consistent-type-imports/